docs/data/data-grid/export/ExcelCustomExport.tsx

```diff
@@ -0,0 +1,326 @@
+    url: '/components/data-grid/export/#excel-export',
+    plan: 'Premium',
+    developed: true,
+  },
+  {
+    id: 26,
+    path: ['Rendering', 'Customizable components'],
+    url: '/components/data-grid/components/)',
+    plan: 'Community',
+    developed: true,
+  },
+  {
+    id: 27,
+    path: ['Rendering', 'Column virtualization'],
+    url: '/components/data-grid/virtualization/#column-virtualization',
+    plan: 'Community',
+    developed: true,
+  },
+  {
+    id: 28,
+    path: ['Rendering', 'Row virtualization > 100 rows'],
+    url: '/components/data-grid/virtualization/#row-virtualization',
+    plan: 'Pro',
+    developed: true,
+  },
+  {
+    id: 29,
+    path: ['Group & Pivot', 'Tree data'],
+    url: '/components/data-grid/group-pivot/#tree-data',
+    plan: 'Pro',
+    developed: true,
+  },
+  {
+    id: 30,
+    path: ['Group & Pivot', 'Master detail'],
+    url: '/components/data-grid/group-pivot/#master-detail',
+    plan: 'Pro',
+    developed: false,
+  },
+  {
+    id: 31,
+    path: ['Group & Pivot', 'Grouping'],
+    url: '/components/data-grid/group-pivot/#grouping',
+    plan: 'Premium',
+    developed: true,
+  },
+  {
+    id: 32,
+    path: ['Group & Pivot', 'Aggregation'],
+    url: '/components/data-grid/group-pivot/#aggregation',
+    plan: 'Premium',
+    developed: false,
+  },
+  {
+    id: 33,
+    path: ['Group & Pivot', 'Pivoting'],
+    url: '/components/data-grid/group-pivot/#pivoting',
+    plan: 'Premium',
+    developed: false,
+  },
+  {
+    id: 34,
+    path: ['Misc', 'Accessibility'],
+    url: '/components/data-grid/accessibility/)',
+    plan: 'Community',
+    developed: true,
+  },
+  {
+    id: 35,
+    path: ['Misc', 'Keyboard navigation'],
+    url: '/components/data-grid/accessibility/#keyboard-navigation',
+    plan: 'Community',
+    developed: true,
+  },
+  {
+    id: 36,
+    path: ['Misc', 'Localization'],
+    url: '/components/data-grid/localization/)',
+    plan: 'Community',
+    developed: true,
+  },
+];
+
+const columns = [
+  {
+    field: 'plan',
+    type: 'singleSelect',
+    valueOptions: ['Community', 'Pro', 'Premium'],
+  },
+  {
+    field: 'developed',
+    type: 'boolean',
+  },
+];
+
+const groupingColDef = {
+  headerName: 'Feature',
+};
+
+const exceljsPreProcess = ({ workbook, worksheet }) => {
+  // Set document meta data
+  workbook.creator = 'MUI-X team';
+  workbook.created = new Date();
+
+  // Customize default excel properties
+  worksheet.properties.defaultRowHeight = 30;
+
+  // Create a custom file header
+  worksheet.mergeCells('A1:C2');
+  worksheet.getCell('A1').value =
+    'This is an helping document for the MUI-X team.\nPlease refer to the doc for up to date data.';
+
+  worksheet.getCell('A1').border = {
+    bottom: { style: 'medium', color: { argb: 'FF007FFF' } },
+  };
+
+  worksheet.getCell('A1').font = {
+    name: 'Arial Black',
+    size: 14,
+  };
+  worksheet.getCell('A1').alignment = { vertical: 'top', horizontal: 'center' };
```

Comment:
```suggestion
  worksheet.getCell('A1').alignment = { vertical: 'top', horizontal: 'center', wrapText: true };
```

Before:
<img width="333" alt="Screenshot 2022-04-26 at 18 45 12" src="https://user-images.githubusercontent.com/13808724/165351376-0363e5bf-1886-425e-961d-d9af8b8db55d.png">
After:
<img width="341" alt="Screenshot 2022-04-26 at 18 45 38" src="https://user-images.githubusercontent.com/13808724/165351369-0811cb0a-5524-4160-a752-ef0f961a6a96.png">